FabNESO/make_sweep_dir.py

```diff
@@ -73,7 +73,7 @@ def main() -> None:
         # Use the dict to create a sweep directory
         create_dict_sweep(
             sweep_path=args.sweep_path,
-            n_divs=args.n_divs,
+            n_dirs=args.n_divs,
```

Comment:
For consistency it might be good to change the name of the command line argument as well to `n_divs` to avoid users getting mixed up between the function and command line interfaces.